homeassistant/components/lamarzocco/__init__.py

```diff
@@ -0,0 +1,59 @@
+"""The La Marzocco integration."""
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import EVENT_HOMEASSISTANT_STOP, Platform
+from homeassistant.core import Event, HomeAssistant
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers.typing import ConfigType
+
+from .const import DOMAIN
+from .coordinator import LmApiCoordinator
+
+PLATFORMS = [
+    Platform.WATER_HEATER,
+]
+
+CONFIG_SCHEMA = cv.config_entry_only_config_schema(DOMAIN)
+
+
+async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
+    """Set up the La Marzocco component."""
+    hass.data.setdefault(DOMAIN, {})
+    return True
```

Comment:
```suggestion
```

Reply:
can we really remove all of that? Isn't line 16 a requirement for all integrations that are only configurable through config flow? I got some checks failing if I don't have that line.

Reply:
You can keep the line but the only thing it will do is create a repair issue if you use yaml.